pkg/languagedetection/languagemodels/language.go

```diff
@@ -30,6 +30,9 @@ const (
 	// PHP language name.
 	PHP LanguageName = "php"
 
+	// CPP language name.
+	CPP LanguageName = "cpp"
```

Comment:
Handling for this should also be added to https://github.com/DataDog/datadog-agent/blob/aa6f673b0f0bd9f2dd47127834b03c1ffa578cf2/pkg/collector/corechecks/servicediscovery/language/language_linux.go#L23 (can be done in a separate PR).